pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/metadata/KafkaMessageMetadata.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.plugin.stream.kafka20.metadata;
+
+import org.apache.pinot.spi.stream.RowMetadata;
+
+
+public class KafkaMessageMetadata implements RowMetadata {
+
+  private final long _ingestionTimeStamp;
+
+  KafkaMessageMetadata(long ingestionTimeStamp) {
+    _ingestionTimeStamp = ingestionTimeStamp;
+  }
+
+  @Override
+  public long getIngestionTimeMs() {
```

Comment:
please keep the name same across getter and field (_ingestionTimeMillis and getIngestionTimeMillis())